test/unit/test-adapter.js

```diff
@@ -146,12 +118,104 @@ describe('SlackMessageAdapter', function () {
 
   describe('#expressMiddleware()', function () {
     beforeEach(function () {
-      this.adapter = new SlackMessageAdapter(workingVerificationToken);
+      this.adapter = new SlackMessageAdapter(workingSigningSecret);
+      this.next = sinon.stub();
+      this.dispatch = sinon.stub();
+      this.res = sinon.stub({
+        setHeader: function () { },
+        end: function () { }
+      });
+      this.errFn = sinon.stub();
     });
+
     it('should return a function', function () {
       var middleware = this.adapter.expressMiddleware();
       assert.isFunction(middleware);
     });
+    it('should error when body parser is used', function () {
+      var middleware = this.adapter.expressMiddleware();
+      var req = { body: { } };
+      var res = this.res;
+      var next = this.next;
+      next.callsFake(function (err) {
+        assert.equal(err.code, errorCodes.BODY_PARSER_NOT_PERMITTED);
+      });
+      middleware(req, res, next);
+    });
+    it('should succeed with basic requests', function (done) {
```

Comment:
while it doesn't hurt to include this test here (it's an integration test), i don't think it belongs here since what its actually testing is functionality implemented in the http-handler. i also think it should have a more specific name, e.g. `should verify correctly signed request bodies`.